emulsify-ds/emulsify-drupal
# feat: update storybook version and return Story() in preview.js

**What:**
Resolves 

**Why:**

_why this change improves emulsify_

**How:**

_more details of changes made_

**To Test:**


Note: The scss files should now compile. However, this will result in a lot of stylint warnings (examples below) but this is more of an issue with the Compound repository and not emulsify-drupal specifically.

```
WARN [stylelint] 
WARN components/style.scss
WARN   2:9  ✖  Expected "'~normalize.css/normalize'" to be "url('~normalize.css/normalize')"                                import-notation
WARN   3:9  ✖  Expected "'~breakpoint-sass/stylesheets/breakpoint'" to be "url('~breakpoint-sass/stylesheets/breakpoint')"  import-notation
WARN   6:9  ✖  Expected "'00-base/**/*.scss'" to be "url('00-base/**/*.scss')"                                              import-notation
WARN   7:9  ✖  Expected "'01-atoms/**/*.scss'" to be "url('01-atoms/**/*.scss')"                                            import-notation
WARN   8:9  ✖  Expected "'02-molecules/**/*.scss'" to be "url('02-molecules/**/*.scss')"                                    import-notation
WARN   9:9  ✖  Expected "'03-organisms/**/*.scss'" to be "url('03-organisms/**/*.scss')"                                    import-notation
WARN  10:9  ✖  Expected "'04-templates/**/*.scss'" to be "url('04-templates/**/*.scss')"                                    import-notation
WARN  11:9  ✖  Expected "'00-base/01-colors/colors-used.scss'" to be "url('00-base/01-colors/colors-used.scss')"            import-notation```
